todo-mvc/src/actions/todoStoreActions.ts

```diff
@@ -19,26 +18,25 @@ export const deleteTodo = createAction({
 
 export const deleteCompleted = createAction({
 	do() {
-		return todoStore.get()
-			// <any> hammer since the store isn't typed to return an iterator (which it does).
-			.then((items: any) => {
-				const promises = Array.from<Item>(items)
-					.filter(({ completed }) => completed)
-					.map(({ id }) => todoStore.delete(id));
-				return Promise.all(promises);
-			});
+		return todoStore.fetch(createFilter().equalTo('completed', true))
+		.then((items: Item[]) => {
```

Comment:
ah yeah sorry that's my bad, I moved to this for debugging and didn't change back